svsim/src/main/scala/Simulation.scala

```diff
@@ -70,7 +89,7 @@ object Simulation {
 
   /** @note Methods in this class and `Simulation.Port` are somewhat lazy in their execution. Specifically, methods returning `Unit` neither flush the command buffer, nor do they actively read from the message buffer. Only commands which return a value will wait to return until the simulation has progressed to the point where the value is available. This can improve performance by essentially enabling batching of both commands and messages. If you want to ensure that all commands have been sent to the simulation executable, you can call `completeInFlightCommands()`.
     */
-  class Controller private[Simulation] (
+  final class Controller private[Simulation] (
```

Comment:
Is this `final` change necessary? I'm going to backport this to 5.x and it may cause a binary compatibility issue. I see the constructor is package private so it's probably okay to waive the binary compatibility issue, but I just wanted to check.

Reply:
It’s not, and I’m annoyed “final” is the default. Have we released this API yet?